src/components/layout/SwapLayout/SwapLayout.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Meta } from '@storybook/react/types-6-0'
+
+import { SwapLayout } from 'components/layout'
+import { RouterDecorator } from 'storybook/RouterDecorator'
+import { LoremIpsum } from 'storybook/LoremIpsum'
+
+export default {
+  title: 'Layout/SwapLayout',
+  component: SwapLayout,
+  decorators: [RouterDecorator],
```

Comment:
What was breaking without? Some `Link`

Reply:
This 😅
With `MemoryRouter` works like as a charm :) 
![image](https://user-images.githubusercontent.com/2352112/93111800-6c9e8e80-f6b7-11ea-95b5-623a8c401ef9.png)